File: Assets/scenes/landinglaunchscene/script/DoorTrigger.js
#pragma strict
var dockChamber : DockChamberScript;


function Start () {

}

function OnTriggerExit(collider : Collider){
	if(collider.name == "TheShip"){
		dockChamber.closeDoor();
		GameObject.Find("AmbientLight").GetComponent.<Light>().enabled = true;
	}
	
}

function Update () {

}

File: Assets/scenes/landinglaunchscene/script/DoorTrigger.cs
using UnityEngine;
using System;


public class DoorTrigger:MonoBehaviour{
	public DockChamberScript dockChamber;
	
	
	public void Start() {
	
	}
	
	public void OnTriggerExit(Collider collider){
		if(collider.name == "TheShip"){
			dockChamber.closeDoor();
		}
		
	}
	
	public void Update() {
	
	}
}
